m4/odp_libconfig.m4

```diff
@@ -0,0 +1,36 @@
+# ODP_LIBCONFIG([ACTION-IF-FOUND], [ACTION-IF-NOT-FOUND])
+# -----------------------------------------------------
+AC_DEFUN([ODP_LIBCONFIG],
+[dnl
+##########################################################################
+# Set optional libconfig path
+##########################################################################
+AC_ARG_WITH([libconfig-path],
+[AS_HELP_STRING([--with-libconfig-path=DIR],
+[path to libconfig libs and headers (use system path if not provided)])],
+[export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$withval/lib/pkgconfig"],
+)
+
+##########################################################################
+# Check for libconfig availability
+##########################################################################
+odp_libconfig_ok=yes
+PKG_CHECK_MODULES([LIBCONFIG], [libconfig], [],
+		  [odp_libconfig_ok=no])
+
+if test "x$odp_libconfig_ok" = "xyes" ; then
+##########################################################################
+# Create a header file odp_libconfig_config.h which containins null
+# terminated hex dump of odp-linux.conf
+##########################################################################
+	[pushd ${srcdir}/config
+	 xxd -i odp-linux.conf | sed 's/\([0-9a-f]\)$/\0, 0x00/' > \
+	   ../platform/${with_platform}/include/odp_libconfig_config.h
+	 popd]
```

Comment:
Please use `AC_CONFIG_COMMANDS` to regenerate this header. Also there should be a dependency line in respective Makefile.am to regenerate the header.

Reply:
Any hints how to do this using AC_CONFIG_COMMANDS?

Reply:
> Also there should be a dependency line in respective Makefile.am to regenerate the header.

Can you elaborate this a bit?

Reply:
Consider somebody changing conf file and reruning make. Make should detect that header should be regenerated and invoke ./config.status with proper tag to regenrate header in question.